### PR TITLE
[FEATURE] Auto create DB fields from TCA for type "input"

### PR DESCRIPTION
**Pull request by linawolf**
Releases: main
Resolves: https://github.com/TYPO3-Documentation/Changelog-To-Doc/issues/937

### CODE REVIEW

File: Documentation/ColumnsConfig/Type/Input/Index.rst [hunk 1]
```diff
@@ -25,6 +25,16 @@ appropriate length.
     *   :php:`renderType=inputLink`: Use the TCA type :ref:`link <columns-link>`
         instead.
 
+..  versionchanged:: 13.2
+    Tables with TCA columns set to `type="input"` do not
+    need an `ext_tables.sql` entry anymore. The Core now
+    creates this field automatically. For short input fields allowing less
+    then 255 chars :sql:`VARCHAR()` is used, :sql:`TEXT` for larger input fields.
```
Comment (fe-hicking) [inline Documentation/ColumnsConfig/Type/Input/Index.rst:32]:
```suggestion
    than 255 chars :sql:`VARCHAR()` is used, :sql:`TEXT` for larger input fields.
```

Comment (fe-hicking) [reply Documentation/ColumnsConfig/Type/Input/Index.rst:32]: We also need to describe this depends on the "max" attribute (cant do myself atm)

File: Documentation/ColumnsConfig/Type/Input/Index.rst [hunk 2]
```diff
@@ -25,6 +25,16 @@ appropriate length.
...
+
+    Extension authors who need or want to override default
+    TCA schema details for whatever reason, can of course
+    do so by defining something specific in ext_tables.sql.
```
Comment (fe-hicking) [inline Documentation/ColumnsConfig/Type/Input/Index.rst:36]:
```suggestion
    do so by defining something specific in :file:`ext_tables.sql`.
```

File: Documentation/ColumnsConfig/Type/Input/Index.rst [hunk 3]
```diff
@@ -25,6 +25,16 @@ appropriate length.
     *   :php:`renderType=inputLink`: Use the TCA type :ref:`link <columns-link>`
         instead.
 
+..  versionchanged:: 13.2
+    Tables with TCA columns set to `type="input"` do not
+    need an `ext_tables.sql` entry anymore. The Core now
```
Comment (garvinhicking) [inline Documentation/ColumnsConfig/Type/Input/Index.rst:30]:
```suggestion
    need an :file:`ext_tables.sql` entry anymore. The Core now
```

File: Documentation/ColumnsConfig/Type/Input/Index.rst [hunk 4]
```diff
@@ -25,6 +25,16 @@ appropriate length.
     *   :php:`renderType=inputLink`: Use the TCA type :ref:`link <columns-link>`
         instead.
 
+..  versionchanged:: 13.2
+    Tables with TCA columns set to `type="input"` do not
+    need an :file:`ext_tables.sql` entry anymore. The Core now
+    creates this field automatically. For short input fields allowing less
```
Comment (froemken) [inline Documentation/ColumnsConfig/Type/Input/Index.rst:31]: Using the word "field" for a field within the TCA form is OK for me, but if we are talking about SQL and table columns I would prefer using "column". The second "field" in that row is OK for me as it relates to the TCE form.

```suggestion
    creates this column automatically. For short input fields allowing less
```

**Review by froemken — CHANGES_REQUESTED** [review]

**Review by froemken — APPROVED** [review]

**Pull request merged**